### PR TITLE
Add reolink.ptz_move service docs

### PR DESCRIPTION
**Pull request by starkillerOG**
## Proposed change

Add reolink.ptz_move service docs, see PR: https://github.com/home-assistant/core/pull/104350

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/104350
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/reolink.markdown [hunk 1]
```diff
@@ -124,6 +124,15 @@ PTZ left, right, up, down, zoom in and zoom out will continually move the camera
 
 "Guard set current position" will set the current position as the new guard position.
 
+### Service reolink.ptz_move
+
+Some Reolink PTZ cameras are capable of moving at different speeds. For those cameras, the `reolink.ptz_move` service can be used in combination with the **PTZ left**, **right**, **up**, **down**, **zoom in** or **zoom out** entity which allows specifying the speed attribute. If the PTZ button entities for a specific camera are not shown under "choose entity" under "targets" of the `reolink.ptz_move` service, it means that camera does not support custom PTZ speeds.
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
Some Reolink <abbr title="pan, tilt, and zoom">PTZ</abbr> cameras can move at different speeds. For those cameras, the `reolink.ptz_move` service can be used in combination with the **PTZ left**, **right**, **up**, **down**, **zoom in**, or **zoom out** entity which allows specifying the speed attribute. If the <abbr title="pan, tilt, and zoom">PTZ</abbr> button entities for a specific camera are not shown under **Choose entity** under **targets** of the `reolink.ptz_move` service, it means that this camera does not support custom <abbr title="pan, tilt, and zoom">PTZ</abbr> speeds.
```

File: source/_integrations/reolink.markdown [hunk 2]
```diff
@@ -124,6 +124,15 @@ PTZ left, right, up, down, zoom in and zoom out will continually move the camera
...
+
+| Service data attribute | Optional | Description                                                                              |
+| ---------------------- | -------- | -----------------------------------------------------------------------------------------|
+| `entity_id`            |      no  | Name of the reolink PTZ button entity to control. For example `button.trackmix_ptz_left` |
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
| `entity_id`            |      no  | Name of the Reolink <abbr title="pan, tilt, and zoom">PTZ</abbr> button entity to control. For example, `button.trackmix_ptz_left`. |
```

File: source/_integrations/reolink.markdown [hunk 3]
```diff
@@ -124,6 +124,15 @@ PTZ left, right, up, down, zoom in and zoom out will continually move the camera
...
+| `speed`                |      no  | PTZ move speed. For example `10`                                                         |
```
Comment (c0ffeeca7) [inline source/_integrations/reolink.markdown]:
```suggestion
| `speed`                |      no  | <abbr title="pan, tilt, and zoom">PTZ</abbr> move speed. For example `10`.                                                         |
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @starkillerOG :+1:

**Comment by starkillerOG** [comment]
Thanks for the tweaks!

**Pull request merged**